modules/administration_manual/pages/maintenance/encryption/migrating-from-user-key-to-master-key.adoc

```diff
@@ -1,6 +1,10 @@
 = Migrating User Key Encryption to Master Key Encryption
 
-If you need to migrate from User Key-based to Master Key-based encryption, there are several steps that you need to follow to ensure a smooth and complete transition. 
+== Why should I move away from User Key-based Encryption
```

Comment:
Can you change this to "Why Should I Move Away From User Key-based Encryption" pls?